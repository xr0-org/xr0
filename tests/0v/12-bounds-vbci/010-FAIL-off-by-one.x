void
access(int *arr) ~ [ setup: arr = .clump(1); ]
{
	arr[1] = 1;
}
