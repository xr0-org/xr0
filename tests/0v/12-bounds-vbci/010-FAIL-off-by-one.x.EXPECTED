12-bounds-vbci/010-FAIL-off-by-one.x:4:12: out of bounds (access)
